css/main_styles.css

```diff
@@ -0,0 +1,224 @@
+* {
+  list-style-type: none;
+  font-family: 'Lato', 'Merriweather', sans-serif;
+}
+
+h2 {
+  text-align: center;
+  margin-top: 3rem;
+  margin-bottom: 2rem;
+}
+
+#container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: auto;
+  width: 95%;
+  margin: auto;
+}
+
+nav {
+  grid-column: 1 / 4;
+  padding-bottom: 0;
+  height: 75px;
+  margin: 2rem;
+}
+
+#nav {
+  height: 100%;
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 1rem;
+}
+
+#nav li {
+  width: 20%;
+}
+
+#nav li:first-child {
+  align-self: flex-start;
```

Comment:
Good job using :fistchild pseudo class to get the first element of your nav list.